src/profile_manager/tests/test_views.py

```diff
@@ -396,6 +401,34 @@ def test_profile_list_staff(self):
         qs = response.context['object_list']
         self.assertEqual(qs.count(), 3)  # self.test_teacher, deck_owner, admin
 
+    def test_profile_list_block__get_queryset(self):
+        """ProfileListBlock view's get queryset method should return a queryset containing only students in the desired block"""
+        self.client.force_login(self.test_teacher)
+
+        # make a new empty block object and get semester + user model to test with
+        testblock = baker.make(Block)
+        current_semester = SiteConfig.get().active_semester
+        user = get_user_model()
```

Comment:
put this at the top of the test file, under the imports, and since it is a model (class) you are getting use capitalized `User`.  See other tests modules for consistency,

Reply:
User capitalized, the siteconfig query was actually redundant, another test already used the active semester variable (shortened to active_sem so it dodged my ctrl + f), I was able to just call that instance instead of getting a new one.